.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,7 +5,9 @@
 #### General
 <!-- You only need to choose one of the first two check items: Generally, test on the test servers. -->
 <!-- If it's only a small change, testing it locally is acceptable and you may remove the first checkmark. If you are unsure, please test on the test servers. -->
-- [ ] I locally pass **all** tests or calculated manually the correct solutions on paper.
+- [ ] I pass **all** artemis tests.
```

Comment:
If we don't have this point, I'm if the makes that much sense anymore because the last two points are same as above

Reply:
> Not sure about this point because you can also write 1 good test but you don't pass all tests on Artemis. As long as the test makes sense and works it is enough for me

It's more about the guarantee level, if pass artemis -> less thought required to see if tests work or not, if not pass artemis -> careful review

Reply:
Alternatively we can keep 'I guarantee correctness' and move the information which tests you pass to another header?

Reply:
How about a general point "correctness" with sub points for passing Artemis, passing the new tests locally and having verified the expected return values manually?
Additionally keep in mind that some exercises allow some different return values/logging. One example will be the second exercise. So passing Artemis alone will not guarantee a good/valid test case

Reply:
> How about a general point "correctness" with sub points for passing Artemis, passing the new tests locally and having verified the expected return values manually? Additionally keep in mind that some exercises allow some different return values/logging. One example will be the second exercise. So passing Artemis alone will not guarantee a good/valid test case

Maybe instead of checkboxes general header 'Correctness of Tests', where PR-owners can tell what was tested in what way, there will always be 'out-of-the-box' way to create tests. A checkbox structure like this could be looked into:

```md
- [x] The tests, if any, are correct
    - [ ] I pass **all** artemis tests.
    - [x] I pass **all** local tests.
    - [x] I calculated manually **all** the correct solutions on paper.
    - [ ] This PR does not modify the test set.
```

Reply:
> ```
> - [ ] I confirm that my changes are correct, helpful and that I pass all local & Artemis tests
> ```
> 
> 
>     
>  
>   
> 
> From point of view is this enough and should be understandable (I said at the beginning that there might the case where you add a good test case but not pass all Artemis tests but I think it's too overkill to find here the perfect solution). I also think we should not overengineer this point because this semester we don't have that much exercises (compared to PGdP) and we have no hidden tests (right?) which means that this repository is not that much important compared to PGdP last semester.

How about your part except after the comma, if you guarantee your tests are correct that’s enough, you don’t have to pass Artemis or the test itself?

Reply:
LGTM